jupyterlab_git/handlers.py

```diff
@@ -391,6 +391,39 @@ async def post(self, path: str = ""):
         self.finish(json.dumps(output))
 
 
+class GitRemoteDetailsShowHandler(GitHandler):
+    """Handler for 'git remote -v'."""
+
+    @tornado.web.authenticated
+    async def get(self, path: str = ""):
+        """GET request handler to retrieve existing remotes."""
+        local_path = self.url2localpath(path)
+        output = await self.git.remote_show(local_path, verbose=True)
+        if output["code"] == 0:
+            self.set_status(201)
```

Comment:
GET should return 200 (OK); 201 (CREATED) is adequate for POST request (see [wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_success))

```suggestion
            self.set_status(200)
```